packages/react-bindings/src/hooks/useStateManager.ts

```diff
@@ -39,11 +39,8 @@ const useStateManager = <
   const latestManager = React.useRef<Manager<State, Actions> | null>(null)
 
   // Heads up! forceUpdate() is used only for triggering rerenders stateManager is SSOT()
-  const [, forceUpdate] = React.useState()
-  const syncState = React.useCallback(
-    (_prevState: State, nextState: State) => forceUpdate(nextState),
-    [],
-  )
+  const [, forceUpdate] = React.useReducer((c: number) => c + 1, 0) as [never, () => void]
```

Comment:
```suggestion
  const [, syncState] = React.useReducer((c: number) => c + 1, 0) as [never, () => void]
```
Or just remove `syncState` and use `forceUpdate`

Reply:
We have only `forceUpdate()` now 👍